cli/mock.go

```diff
@@ -1,79 +0,0 @@
-package cli
-
-import (
-	"context"
-	"errors"
-
-	"github.com/spf13/cobra"
-	"go.keploy.io/server/v2/config"
-	recordSvc "go.keploy.io/server/v2/pkg/service/record"
-	replaySvc "go.keploy.io/server/v2/pkg/service/replay"
-	"go.keploy.io/server/v2/utils"
-	"go.uber.org/zap"
-)
-
-func init() {
-	Register("mock", Mock)
-}
-
-func Mock(ctx context.Context, logger *zap.Logger, _ *config.Config, serviceFactory ServiceFactory, cmdConfigurator CmdConfigurator) *cobra.Command {
```

Comment:
@gouravkrosx why are we deleting this?

Reply:
This command was not in use. I had a word with @charankamarapu also.

Reply:
cool